model/config.go

```diff
@@ -2860,7 +2860,7 @@ func (mes *MessageExportSettings) isValid(fs FileSettings) *AppError {
 
 func (ds *DisplaySettings) isValid() *AppError {
 	if len(ds.CustomUrlSchemes) != 0 {
-		validProtocolPattern := regexp.MustCompile(`(?i)^\s*[a-z][a-z0-9-]*\s*$`)
+		validProtocolPattern := regexp.MustCompile(`(?i)^\s*[a-z][a-z0-9-]*([+.][a-z]+)?\s*$`)
```

Comment:
Related comments on https://github.com/mattermost/marked/pull/11, but I think we'll want to tweak the regular expression here as discussed on that PR. (I'm guessing the server regex doesn't support the unicode extensions, but the placement and repetition still applies I believe.)

Reply:
Thanks for the review, I just replied here https://github.com/mattermost/marked/pull/11